src/utils/file.py

```diff
@@ -44,6 +46,11 @@ def setup_dirs_for_paths(paths):
             logger.info(f"Created : {save_output_dir}")
             os.makedirs(save_output_dir)
 
+    for save_output_dir in [paths.evaluation_dir]:
+        if os.path.exists(save_output_dir):
+            shutil.rmtree(save_output_dir)
```

Comment:
let's not remove any of the user's output. Conditionally create the directory instead similar to other lines here